WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt

```diff
@@ -176,6 +163,42 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
                 }
             }
         }
+
+        @ExperimentalStdlibApi
```

Comment:
Just a note: This logic would ideally be located in the VM (and related classes). However, this refactoring is above the scope of this PR.

Reply:
Agreed - I went back and forth on it myself. Good next hackweek project to clean this viewholder up.